x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/layer/compatible_layer.tsx

```diff
@@ -0,0 +1,432 @@
+
+              <EuiSpacer size="s" />
+              <EuiFormRow>
+                <EuiCheckbox
+                  id="realTime"
+                  disabled={startJob === false}
+                  data-test-subj={`mlLensLayerRealTimeCheckbox_${layerIndex}`}
+                  checked={runInRealTime}
+                  onChange={(e) => setRunInRealTime(e.target.checked)}
+                  label={i18n.translate(
+                    'xpack.ml.embeddables.lensLayerFlyout.createJobCallout.additionalSettings.realTime',
+                    {
+                      defaultMessage: 'Leave the job running for new data',
+                    }
+                  )}
+                />
+              </EuiFormRow>
+              <EuiSpacer size="m" />
+            </EuiAccordion>
+          </EuiForm>
+          <EuiSpacer size="m" />
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiButton
+                disabled={
+                  state === STATE.VALIDATING ||
+                  jobId === '' ||
+                  jobIdValid !== '' ||
+                  bucketSpanValid !== ''
+                }
+                onClick={createADJob.bind(null, layerIndex)}
+                size="s"
+                data-test-subj={`mlLensLayerCreateJobButton_${layerIndex}`}
+              >
+                <FormattedMessage
+                  id="xpack.ml.embeddables.lensLayerFlyout.createJobButton.saving"
+                  defaultMessage="Create job"
+                />
+              </EuiButton>
+            </EuiFlexItem>
+
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
+                onClick={createADJobInWizard.bind(null, layerIndex)}
+                size="s"
+                iconType="popout"
+                iconSide="right"
+                data-test-subj={`mlLensLayerCreateWithWizardButton_${layerIndex}`}
+              >
+                <FormattedMessage
+                  id="xpack.ml.embeddables.lensLayerFlyout.createJobButton"
+                  defaultMessage="Create job using wizard"
+                />
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </>
+      ) : null}
+
+      {state === STATE.SAVE_SUCCESS ? (
+        <>
+          <EuiFlexGroup
+            gutterSize="s"
+            data-test-subj={`mlLensLayerCompatible.jobCreated.success_${layerIndex}`}
+          >
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">
+                <EuiIcon type="checkInCircleFilled" color="success" />
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText size="s">
+                <FormattedMessage
+                  id="xpack.ml.embeddables.lensLayerFlyout.saveSuccess"
+                  defaultMessage="Job created"
+                />
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+
+          <EuiSpacer size="s" />
+          <EuiButtonEmpty
```

Comment:
I don't think it makes sense to show the links to view the results for jobs that aren't started. What about showing a link to the ML jobs list instead if the job isn't started?

Reply:
Updated in https://github.com/elastic/kibana/pull/136421/commits/d75dcc12f9154c7cb992c32de9b642b3f9cd25dc